src/persist-client/src/internal/apply.rs

```diff
@@ -506,7 +508,20 @@ where
             );
         }
 
-        let write_rollup = new_state.need_rollup(ROLLUP_THRESHOLD.get(cfg));
+        let now = (cfg.now)();
+        let write_rollup = new_state.need_rollup(
+            ROLLUP_THRESHOLD.get(cfg),
+            ROLLUP_USE_ACTIVE_ROLLUP.get(cfg),
+            u64::cast_from(ROLLUP_FALLBACK_THRESHOLD_MS.get(cfg)),
+            now,
+        );
+
+        if write_rollup.is_some() && ROLLUP_USE_ACTIVE_ROLLUP.get(cfg) {
+            new_state.collections.active_rollup = Some(ActiveRollup {
+                seqno: state.seqno,
```

Comment:
Nit: let's use the seqno contained in the `write_rollup` here instead of the current seqno. (Currently they're the same, but that way they stay in sync.)